exercises/practice/markdown/markdown.py

```diff
@@ -1,7 +1,13 @@
 import re
 
-
 def parse(markdown):
+
+    """ Markup
+    Input: would be a markdown phrase for an example
+    Output: expected to pass a HTML equivalent for the same
+    Sample: "#exercism.org is a useful practice ground"
+    you syntax should return <h1>exercism.org is a useful practice ground </h1>
+    """
```

Comment:
```suggestion
    """Refactor this code.
    
    The intent of this exercise is to have you practice a refactor of the code in this file
    to increase its readability and maintainability.
    
    We suggest you make notes for reviewers as you do so, 
    to give them a starting point for discussion.
    """
```

This is the same suggestion as @BethanyG gave (though to be fair, she did say "something like" and that suggested a bit more creativity could be involved than probably was wanted).

This is would be preferable, though I will defer to you as to the indentation and particular placement.

Reply:
Yes, I have made it more readable and have added context to it.